libraries/AC_InputManager/AC_InputManager_Heli.cpp

```diff
@@ -107,17 +107,34 @@ float AC_InputManager_Heli::get_pilot_desired_collective(int16_t control_in)
     }
     acro_col_out = constrain_float(acro_col_out, 0.0f, 1.0f);
 
-    // ramp to and from stab col over 1/2 second
-    if (_im_flags_heli.use_stab_col && (_stab_col_ramp < 1.0f)){
-        _stab_col_ramp += 2.0f/(float)_loop_rate;
-    } else if(!_im_flags_heli.use_stab_col && (_stab_col_ramp > 0.0f)){
-        _stab_col_ramp -= 2.0f/(float)_loop_rate;
+    if (init_transition_ramp) {
```

Comment:
I think you could delete this init_transition_ramp bool.  You are already setting _ramp to 1.0.  at the same point that you set init_transisition_ramp.  See my comment later on about using is_positive(_ramp) instead.